### DISCUSSION TITLE
Fluid audio visualiser

### DISCUSSION THREAD

**Discussion by StickyBoy321** [rocksdanister/lively#705] [Q&A]
What is the code used to detect desktop sounds for the fluid audio visualiser wallpaper?
Ps. Sorry I've never really coded anything before so I feel dumb lol

**Comment by rocksdanister** [comment]
Its a webpage and Lively is sending the audio data:
https://github.com/rocksdanister/lively/wiki/Web-Guide-II-:-System-Audio-Data
